packages/Popover/src/positionUtils.spec.js

```diff
@@ -0,0 +1,274 @@
+import { Align, Justify } from './Popover';
+import { calculatePosition, getElementPosition } from './positionUtils';
+
+const SPACING = 10;
+const CENTERED_REFELPOS = {
```

Comment:
General nit, can we use whole numbers for all values? Is there a reason to use decimals? Will whole numbers allow us to still test things as require?

Reply:
Also perhaps we can restructure this:

```
const refElPos = {
  centered: {},
  topLeft: {}
}
```

Also generally I try and reserve all uppercase for string constants. These are just objects that we're using, so no need to uppercase them.

Reply:
The reason that I'm using decimals is because I'm testing in the browser as well to make sure these values appear not just in the calculations but also IRL and in the browser there end up being decimals. I can just do pure calculations without double-checking the output elsewhere, if you think that's cleaner and that the double-check is unnecessary.